### DISCUSSION TITLE
Adding satellite imagery products

### DISCUSSION THREAD

**Discussion by davidpodeur** [wmo-im/wis2box#622] [Q&A]
Are there any resources for non synop/temp data sharing through wis2box ?

How could be publish for example satellite imagery ?
=> Which metadata-satellite.yml could be used ?
=> Which topics are reserved for this use ?

Thanks in advance for the feedback.

David

**Comment by maaikelimper** [comment]
Hi David,

Thank you so much for your question and your interest in wis2box.

You not the only user asking for guidance on topics and metadata-templates for non synop/temp data sharing through wis2box !

I acknowledge that right now wis2box-repository lacks these resources. 

You can check the following repositories that are responsible for WIS2 Metadata and wis2-topic-hierarchy respectively:
https://github.com/wmo-im/wcmp2
https://github.com/wmo-im/wis2-topic-hierarchy

However as documents describing the WIS2 Architecture are currently being prepared for INFCOM, these might not be up to date.

I will follow-up with the WIS2 Architecture and WIS2 Metadata team to ensure that wis2box can reference guidelines for topics and metadata for commonly used data-types, including satellite imagery.

**Comment by maaikelimper** [comment]
@golfvert FYI
@tomkralidis is there a metadata example for @davidpodeur ?

**Comment by tomkralidis** [comment]
Bonjour @davidpodeur:

- WCMP2 examples can be found in https://github.com/wmo-im/wcmp2/blob/main/examples/ (EUMETSAT example in https://github.com/wmo-im/wcmp2/blob/main/examples/int-eumetsat-serviri-core.json)
- satellite topics are yet to be officially defined at the moment; do you have an example description of a dataset sample we can assess?  Note: this could be put forth as an official WCMP2 example as well
